wiki/sustainable_distance_education.qmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Sustainable Distance Education"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Elena Scheid"
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+K (key quote):
+
+
+"An independent model may hamper students’ perception of
+current knowledge and development of collaborative competences, in contrast to the more-interactive
+collaborative and social models" (p. 10).
+
+---
+
+A (main argument):
+
+Generally, only a small influence from the particular learning model on the students’ perception of knowledge acquisition was being observed. 
+The empirical research in this study couldn't notice that the individual model is more useful to gain technical and theoretical knowledge.
+In fact, the other two models - the social discussion model and the collaborative learning model - seem to cover this better (compare p. 10).
+
+---
+
+R (relation to arguments in other articles):
+
+Similar studies also mark positive learning experiences with digital pedagogy in digital environments. But they often did not do deep and detailed 
+research by comparing different digital learning models to really measure the effects from the digital surrounding on self-learning-experiences. 
+They rather focuse on the role and importance of collaborative online communities less than autonomous learning.
+
+---
+
+Q (questions):
+
+As the independant model "proved itself generally effective" (p. 11), it could be interesting to analyze its usefulness in certain cases.
+
+1. How valuable is the independant model when it comes to environmental problems and topics?
```

Comment:
Try to name some arguments why the model might be well or not well suited to explain learning with regard to these topics and try to connect your argument with the associated autonomy need experiences.